app/claim/route.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  unauthenticatedRedirect: 'signup',
+
+  model: function(params){
+    return {
+      verificationCode: params.verification_code,
+      invitationId: params.invitation_id,
+      type: 'invitation'
+    };
+  },
+
+  actions: {
+    didTransition: function(){
+      var options = this.currentModel;
+      var verification = this.store.createRecord('verification', options);
+
+      verification.save().then( () => {
+        this.replaceWith('index');
+      }, () => {
+        this.controllerFor('claim').set('error', `
+          There was an error accepting this invitation. Perhaps
+          the verification code has expired?
+        `);
```

Comment:
There isn't anything to click on, the layout is akin to a login box.

The goal is to show some content on the page which the verification is happening. To show a box letting the user know something is going on.

As I've gone though the passwords flow, I started thinking about this UX more. IMO, at the least the user should see what their current email address is before they accept an invitation. This would help protect from automatic acceptance of an invite when a user clicks on a link.

In fact now that I think of it more, this is technically an XSS attack 0_o. You could make an arbitrary user accept an invite just by linking them, or even opening the URL in an iframe.

So, this flow should really be replaced with a button.  @sandersonet if that sounds good to you I can refactor this to have a box prompting the user with whatever easy data we already have for now. At least their current email.